recipe/meta.yaml

```diff
@@ -1,44 +1,45 @@
 {% set version = "1.0.0" %}
 
 package:
-    name: gridfill
-    version: {{ version }}
+  name: gridfill
+  version: {{ version }}
 
 source:
-    fn: v{{ version }}.tar.gz
-    url: https://github.com/ajdawson/gridfill/archive/v{{ version }}.tar.gz
-    md5: 7258701d8296f974268e634ea09ed673
+  url: https://github.com/ajdawson/gridfill/archive/v{{ version }}.tar.gz
+  sha256: e56a02078d3827535a419250256d2b906eb03b00c40045df3d4020fb1b2a04b3
 
 build:
-    number: 0
-    script:
-        - python setup.py install --single-version-externally-managed --record record.txt  # [not (win and py<35)]
-        - python setup.py build --compiler=mingw32  # [win and py<35]
-        - python setup.py install  # [win and py<35]
+  number: 1
+  skip: True  # [win and py2k]
+  script:
+    - python -m pip install . --no-deps --ignore-installed
 
 requirements:
-    build:
-        - python
-        - setuptools
-        - cython
-        - numpy x.x
-        - mingwpy  # [win and py<35]
-    run:
-        - python
-        - numpy x.x
+  build:
+    - python
+    - pip
+    - cython
+    - numpy 1.11.*
+  run:
+    - python
+    - numpy >=1.11,<1.12
```

Comment:
Why is the version of numpy pinned like this, I'm guessing the unfixed numpy deprecations? I am fixing this in source so we can do a new release that has better compatibility.

Reply:
Yep. The tests do not pass on `numpy 1.12`. We can do `>=1.11` once those are fixed.

Reply:
I've released a new version compatible with latest numpy. I'm in the process of updating the recipe, PR inbound.